src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -16,4 +16,5 @@
     public static final Prefix PREFIX_FIELD = new Prefix("f/");
     public static final Prefix PREFIX_APPLICATION_STATUS = new Prefix("as/");
     public static final Prefix PREFIX_INTERVIEW_STATUS = new Prefix("is/");
+    public static final Prefix PREFIX_AVAILABILITY = new Prefix("avail/");
```

Comment:
I know we discussed it to be `avail/`.

However, was thinking maybe we could use `free/` instead. Though it's just 1 character shorter but it has a more straightforward meaning than `avail/` when the user looks at that.

To do it in v1.3. Let's discuss En En! 😄 

Regardless, really great job with this functionality! 👍🏻 💯

Reply:
It is possible, although the prefix is usually an abbreviation of the actual word. 🤔 
But up for discussion as well!